src/components/QuizQuestion/QuizQuestion.jsx

```diff
@@ -2,6 +2,8 @@ import React from "react";
 import { Button, Flex, Text, useMantineTheme } from "@mantine/core";
 import PropTypes from "prop-types";
 
+import { ProgressBar } from "../ProgressBar";
```

Comment:
`import { ProgressBar } from "@/components";`

Reply:
Changed it